wallet/wallet.go

```diff
@@ -2272,29 +2272,6 @@ func (w *Wallet) AccountXpriv(ctx context.Context, account uint32) (*hdkeychain.
 	return privKey, nil
 }
 
-// TxBlock returns the hash and height of a block which mines a transaction.
-func (w *Wallet) TxBlock(ctx context.Context, hash *chainhash.Hash) (chainhash.Hash, int32, error) {
```

Comment:
probably want to keep this, it's not exclusive to tickets.

Reply:
Restored that func in a new commit. Will squash all commits before merge.

Reply:
lgtm